README.md

```diff
@@ -63,7 +61,7 @@ ys1 = NTuple{4,Float32}(ys)
 y2 = ys[2]   # getindex
 
 # Update one element of a vector:
-ys = setindex(ys, 5, 3)   # cannot use ys[3] = 5
+ys = Base.setindex(ys, 5, 3)   # cannot use ys[3] = 5
```

Comment:
There is no `setindex` in `Base`, there is only a `setindex!`.

Reply:
```julia
julia> Base.setindex
setindex (generic function with 2 methods)

julia> methods(Base.setindex)
# 2 methods for generic function "setindex":
[1] setindex(x::Tuple, v, i::Integer) in Base at tuple.jl:45
[2] setindex(nt::NamedTuple, v, idx::Symbol) in Base at namedtuple.jl:325

julia> versioninfo()
Julia Version 1.5.0-DEV.255
Commit dd426ba* (2020-02-09 18:28 UTC)
```

Reply:
I think there is one actually.

```
julia> Base.setindex
setindex (generic function with 2 methods)
```

Reply:
Oops, I was confused.